label_sync/labels.md

```diff
@@ -0,0 +1,124 @@
+| <a id="status/in-progress" href="#status/in-progress">`status/in-progress`</a> | Used during release burndown. Denotes that an issue is actively being worked.| [@kubernetes/kubernetes-milestone-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members) |  [milestonestatus](https://git.k8s.io/test-infra/prow/plugins/milestonestatus) |
+| <a id="status/in-review" href="#status/in-review">`status/in-review`</a> | Used during release burndown. Denotes that a fix for an issue is actively being reviewed.| [@kubernetes/kubernetes-milestone-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members) |  [milestonestatus](https://git.k8s.io/test-infra/prow/plugins/milestonestatus) |
+
+## Labels that apply to issues
+
+| Name | Description | Added By | Prow Plugin |
+| ---- | ----------- | -------- | --- |
+| <a id="help wanted" href="#help wanted">`help wanted`</a> | Denotes an issue ready for a new contributor according to the [help-wanted] guidelines, applied in response to /help by the help prow plugin. <br><br> This was previously `help-wanted`, `for-new-contributors`, | anyone |  [help](https://git.k8s.io/test-infra/prow/plugins/help) |
+| <a id="needs-sig" href="#needs-sig">`needs-sig`</a> | Indicates an issue lacks a `sig/foo` label and requires one. The label will be removed when a `sig/` label is applied.| prow |  [requiresig](https://git.k8s.io/test-infra/prow/plugins/requiresig) |
+
+## Labels that apply to prs
+
+| Name | Description | Added By | Prow Plugin |
+| ---- | ----------- | -------- | --- |
+| <a id="approved" href="#approved">`approved`</a> | Indicates a PR has been approved by an approver from all required OWNERS files.| approvers |  [approve](https://git.k8s.io/test-infra/prow/plugins/approve) |
+| <a id="cherrypick-approved" href="#cherrypick-approved">`cherrypick-approved`</a> | Indicates a cherrypick PR into a release branch has been approved by the release branch manager. Consumed by the kubernetes/kubernetes cherrypick-queue.| humans | |
+| <a id="cherrypick-candidate" href="#cherrypick-candidate">`cherrypick-candidate`</a> | Denotes a PR to master as a candidate for cherry picking into a release branch.| humans | |
+| <a id="cla  human-approved" href="#cla  human-approved">`cla: human-approved`</a> | DEPRECATED. This is a holdover from when we had difficulties getting either the google or cncf CLA bots to behave. It can only be manually applied and is consumed by the submit-queue. It has seen very little use lately.| humans | |
```

Comment:
Does tide listen to this? Should we deprecate it?

Reply:
I'd like to address in followup PR so as not to change content in this PR.  I think the reason I haven't removed this yet is because I saw a recent case where this was used.  But no, tide doesn't use it.